### ISSUE TITLE
Remove partially unsupported wizard options

### ISSUE THREAD

**Issue by Lms24** [getsentry/sentry-wizard#491]
According to our documentation (e.g.[Readme](https://github.com/getsentry/sentry-wizard#options)), the wizard supports a few options across the individual integration flows. This however is no longer true for the modern, clack-based wizard flows (Next, Sveltekit, Remix, ReactNative, Sourcemaps). 

I would like to remove old baggage by dropping wizard options in a major release:

* `--quiet` IMO there's no need for a quiet flow, given that wizards are usually executed locally by users
* `--signup` The new login flow asks if users already have a Sentry account. Way more intuitive than specifying this via an option

We still have a bunch of older flows that theoretically support these options. Ideally, we find a way to continue support there. Maybe, `yargs` has an option for this.

**Comment by mydea** [comment]
is this still relevant/true @lms24

**Comment by Lms24** [comment]
Yes, still relevant. I could just cut a major and remove them. Probably also a good opportunity to remove the uninstall flow from some of the few wizards that "support" it 🤔

**Comment by Lms24** [comment]
Going to close this issue since we de-prioritized all feature work related to the wizard in favour of the AI-based `sentry init` SDK setup.
